### PR TITLE
Collect core dump URLs from dumpling service to raise up to CI.

### PR DESCRIPTION
**Pull request by bryanar**
This change causes the URL of the uploaded dump to be placed in a local_dumplings.txt file that is local to the test.  After the tests have executed, we merge all of the produced local_dumplings.txt and save the file in $(PWD)\dumplings.txt. A Jenkins plugin then reads the contents of the file and shows them.

### CODE REVIEW

File: tests/runtest.sh [hunk 1]
```diff
@@ -116,6 +116,13 @@ case $OSName in
         ;;
 esac
 
+# clean up any existing dumpling remnants from previous runs.
+dumplingsListPath="$(PWD)/dumplings.txt"
```
Comment (bryanar) [inline tests/runtest.sh]: Consider only doing this if dump collection is enabled.

Comment (adityamandaleeka) [inline tests/runtest.sh]: Why is PWD capitalized?

**Comment by adityamandaleeka** [comment]
It looks like you're no longer printing the dumpling ID or the URL that it was uploaded to in the console output (at least I don't see them in there). Can you make sure those are left in? It's really helpful to have them in the console output when investigating failures.

**Comment by mmitche** [comment]
I fixed this this morning.

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test

**Comment by bryanar** [comment]
@mmitche 
http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/checked_ubuntu_flow_prtest/4620/

**Pull request merged**